snowpack/src/build/import-css.ts

```diff
@@ -0,0 +1,39 @@
+import postCss from 'postcss';
+import postCssModules from 'postcss-modules';
+import {logger} from '../logger';
+
+const cssModuleNames = new Map<string, string>();
```

Comment:
The third step is to keep existing behavior. How do we store the JSON response that CSS Modules generate? Before it was kept in the proxy response, but now that it’s centralized it has to live somewhere.

We do now build a `.module.css.json` file with this info, which we could use in the proxy file. That may be correct, but would technically be a change in behavior for existing users. So instead, we simply keep the JSON output in memory until it‘s needed. This memory gets blown away after every build. And in the dev server, it only keeps requested CSS Modules in memory, and doesn’t leak with HMR. It shouldn’t cause any issues for Snowpack users, and it keeps existing behavior as-is, which is important.